### ISSUE TITLE
Analizar funcion de FlowInit

### ISSUE THREAD

**Issue by ekiim** [ekiim/fakemoney#93]
En los archivos JavaScript, hay una función que inicia los flujos, hay que tratar de generalizar esa función para poder tener flujos a lo largo de toda la implementación.

**Comment by Maricruz-GML** [comment]
Se requiere de una investigacion?

Encontre este link con un poco de informacion al respecto:  https://flow.org/en/docs/usage/

**Comment by ekiim** [comment]
No, `FlowInit` es una pequeña colección de funciones que escribí en el archivo `www/js/w3-ext.js` 

Eso de `flow.org` es otra cosa totalmente ajena a nosotros.
